moveit_core/kinematic_constraints/src/utils.cpp

```diff
@@ -227,6 +246,22 @@ moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_n
   return goal;
 }
 
+bool updatePoseConstraint(moveit_msgs::msg::Constraints& constraints, const std::string& link_name,
+                          const geometry_msgs::msg::PoseStamped& pose)
+{
+  // Convert message types so the existing functions can be used
+  geometry_msgs::msg::PointStamped point;
```

Comment:
@AndyZe I don't know why I missed this, but I think we have to pass the stamp information with this as well, since that one is defining the reference frame

Reply:
https://github.com/ros-planning/moveit2/pull/1890

Reply:
https://github.com/ros-planning/moveit2/pull/1890